DSCResourcesTestsHelper/DSCResourcesTestsHelper.psd1

```diff
@@ -0,0 +1,58 @@
+@{
+# Version number of this module.
+ModuleVersion = '1.10.0.0'
```

Comment:
all comments to the DSCResourceCommonTests.psd1 apply here.
In addition, rename this to DSCResourceTestHelper.psd1 (same for module) - let's use singular here to be consistent with the repo name